ui/index.jsx

```diff
@@ -117,6 +117,16 @@ ipcRenderer.on('open-uri-requested', (event, url, newSession) => {
   handleError();
 });
 
+autoUpdater.on('update-available', ({ version }) => {
```

Comment:
were you able to get and store the release notes too? It would help to be able to display those.

Reply:
https://github.com/lbryio/lbry-desktop/issues/7362 could knock this one out on the way. prereleases being ready for update still shows release notes of the last release.

Reply:
Hello @jessopb, I just sent a new update where now we use the `releaseNotes` from the event. Please, let me know if it's working fine on your end. Thanks!